CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 ### What's new in 1.15.0
 * Add support for dependent extensions
-* Fixed bugs under June Milestone
+* [Fixed issues] https://github.com/microsoft/vscode-mssql/milestone/22
```

Comment:
You need the parens as well (this is markdown link syntax - you should verify that when previewing the rendered content it shows up as a link)

And you missed the closed param too

`[Fixed issues](https://github.com/microsoft/vscode-mssql/milestone/22?closed=1)`